third_party/move/move-model/src/model.rs

```diff
@@ -876,6 +919,7 @@ impl GlobalEnv {
         Loc {
```

Comment:
Is there a reason not to use `Loc::new` here?

Reply:
It was already written that way.  But I've updated it.